tests/nest_codegenerator_test.py

```diff
@@ -109,6 +110,38 @@ def test_iaf_cond_alpha_functional(self):
         nestCodeGenerator = NESTCodeGenerator()
         nestCodeGenerator.generate_code(iaf_cond_alpha_functional)
 
+    def test_iaf_psc_alpha_with_codegen_opts(self):
+        input_path = str(os.path.realpath(os.path.join(os.path.dirname(__file__), os.path.join(
+            os.pardir, 'models', 'iaf_psc_alpha.nestml'))))
+
+        code_opts_path = str(os.path.realpath(os.path.join(os.path.dirname(__file__),
```

Comment:
These codegen_opts are written to a JSON file, but are they then actually passed to NESTML?

Reply:
Done.